game/src/index.ts

```diff
@@ -47,6 +53,11 @@ type WritePayload = {
   answer: string;
 };
 
+type VotePayload = {
+  userId: string;
+  answer: string;
```

Comment:
```suggestion
  categoryIndex: number;
  points: number;
```

altough it's fine to keep the `points` for a later PR.

Reply:
Is `categoryIndex` the index of the answer in the category?

Reply:
Whoops disregard, other comment referring to this was hidden for some reason

Reply:
You're right to be confused I think we also do need the `userId` in the payload, but it's the user id of the "voteD" player, not the voter! This is confusing 😅